open-machine-learning-jupyter-book/assignments/data-science/matplotlib-applied.ipynb

```diff
@@ -0,0 +1,406 @@
+{
```

Comment:
This file should be in the `assignments/data-science/` folder.

Reply:
I think I have already put this file in the assignments/data-science/folder, i don't know where the problem is.